content/10-managing-databases/10-storing-large-data.mdx

```diff
@@ -0,0 +1,57 @@
+
+<PrismaOutlinks>
+
+The [Prisma Data Platform](https://www.prisma.io/dataplatform) can help simplify access to your database in production environments.  If you are using [Prisma Client](https://www.prisma.io/docs/concepts/components/prisma-client) to manage your database connections, the Prisma Data Platform may help you manage your production workloads more easily.
+
+</PrismaOutlinks>
+
+## What is object-based storage?
+Object storage, is a storage strategy that manages and manipulates data storage as distinct units, called objects. Each individual object is the combination of the pieces of data that make up a file. The object includes all of the relevant metadata of that file, and then attaches a custom identifier for reference. Object metadata typically identifies the properties of the object, how the object should be handled when accessed, and other application-specific or user-specific information. Each object is then assigned a uniquely identifiable URL path that can be referenced by an application or web server to access that file. This is then what is stored in the database taking up much less space and mitigating other pitfalls we’ll discuss.
+
+Object stores can hold any type of data spanning from human-readable files to unstructured data files (binary files) to media files such as audio, video, or image.  For the purposes of this guide, we will focus on object stores in relation to databases and how they are a better fit for these aforementioned large data types. 
+
+## What are the pitfalls of storing large data in a database?
+To be clear, it is possible to store large data such as text or image files in a database as you would any other piece of information. However, this viability quickly runs dry as your database scale grows. On a large scale database, storing these file types will lead to major pitfalls that are better to avoid. 
+
+### Slower database queries
+Query performance depends a considerable amount on the database’s ability to use database resources to serve the query response. When including files in the database,  overall query performance becomes slower simply from more transmitted data. The files are large and can take up considerable RAM resources. RAM is a finite database resource that also acts as memory for recently accessed data. If queries accessing large data are stored in RAM, then there is limited RAM now available for additional queries potentially forcing the data to be read from disk. Any time data has to come from disk usually results in slower performance. 
```

Comment:
```suggestion
Query performance depends a considerable amount on the database’s ability to use database resources to serve the query response. When including files in the database,  overall query performance becomes slower simply from more transmitted data. The files are large and can take up considerable RAM. RAM is a finite resource that also acts as a cache for recently accessed data. If queries are using the system's RAM to handle large data objects, then there is limited RAM now available for additional queries, potentially forcing the data to be read from disk. Any time the database has to retrieve data from disk results in slower performance.
```

Feel free to disregard the specifics of these edits. I'm just trying to make it flow easier in my head.